packages/js/src/analysis/plugins/replacevar-plugin.js

```diff
@@ -65,19 +65,25 @@ var YoastReplaceVarPlugin = function( app, store ) {
  * @returns {void}
  */
 YoastReplaceVarPlugin.prototype.registerReplacements = function() {
-	this.addReplacement( new ReplaceVar( "%%currentdate%%",     "currentdate" ) );
-	this.addReplacement( new ReplaceVar( "%%currentday%%",      "currentday" ) );
-	this.addReplacement( new ReplaceVar( "%%currentmonth%%",    "currentmonth" ) );
-	this.addReplacement( new ReplaceVar( "%%currenttime%%",     "currenttime" ) );
-	this.addReplacement( new ReplaceVar( "%%currentyear%%",     "currentyear" ) );
-	this.addReplacement( new ReplaceVar( "%%date%%",            "date" ) );
-	this.addReplacement( new ReplaceVar( "%%userid%%",          "userid" ) );
-	this.addReplacement( new ReplaceVar( "%%id%%",              "id" ) );
-	this.addReplacement( new ReplaceVar( "%%page%%",            "page" ) );
-	this.addReplacement( new ReplaceVar( "%%searchphrase%%",    "searchphrase" ) );
-	this.addReplacement( new ReplaceVar( "%%sitedesc%%",        "sitedesc" ) );
-	this.addReplacement( new ReplaceVar( "%%sitename%%",        "sitename" ) );
-	this.addReplacement( new ReplaceVar( "%%category%%",        "category" ) );
+	this.addReplacement( new ReplaceVar( "%%currentdate%%",       "currentdate" ) );
+	this.addReplacement( new ReplaceVar( "%%currentday%%",        "currentday" ) );
+	this.addReplacement( new ReplaceVar( "%%currentmonth%%",      "currentmonth" ) );
+	this.addReplacement( new ReplaceVar( "%%currenttime%%",       "currenttime" ) );
+	this.addReplacement( new ReplaceVar( "%%currentyear%%",       "currentyear" ) );
+	this.addReplacement( new ReplaceVar( "%%post_year%%",         "post_year" ) );
+	this.addReplacement( new ReplaceVar( "%%post_month%%",        "post_month" ) );
+	this.addReplacement( new ReplaceVar( "%%author_first_name%%", "author_first_name" ) );
+	this.addReplacement( new ReplaceVar( "%%author_last_name%%",  "author_last_name" ) );
+	this.addReplacement( new ReplaceVar( "%%taxonomy_title%%",    "taxonomy_title" ) );
```

Comment:
`taxonomy_title` is not implemented on this PR, so I think it should be removed from here too.